### ISSUE TITLE
Unwanted bs-tooltip behavior on iOS

### ISSUE THREAD

**Issue by danidr** [kaliber5/ember-bootstrap#292]
Hello there,
I have found a bug (which is actually a bug in the bootstrap implementation of tooltip) which could be fixed, though, in your Ember plug-in.
On iOS devices, both the hover and focus events are simulated when a user taps on any element. Therefore, when adding a bs-tooltip to any button or link, the user will need to double-tap on such element in order to trigger the action, because on the first tap, the tooltip will be triggered.
By checking whether browser is touch-enabled or not whenever catching the hover or focus event, though I believe this behavior could be prevented, by bubbling the event to the containing component.

To sum it up, on iOS device:
User tap > Registered as hover (tooltip appears) > User tap > Registered as click (trigger action)

While the expected behavior should be:
User tap > Registered as hover > Tooltip component bubbles action > Registered as click (trigger action)
Whaddaya think?

**Comment by danidr** [comment]
@acorncom Actually, no... it still has the same behavior, needing a double tap since the first one only triggers the tooltip, and the second one finally "goes through".

**Comment by jelhan** [comment]
Is this still an issue in recent versions of Ember Bootstrap?
